package.json

```diff
@@ -45,6 +45,7 @@
     "webpack-inline-svg-plugin": "^1.1.4"
   },
   "dependencies": {
+    "mermaid": "^9.4.0",
```

Comment:
I have tried changing the version to "^10.0.0" as well, as was suggested in the upstream PR. This didn't work for me as well.

Reply:
I believe it requires a newer version of webpack/html-webpack-plugin, which themselves require new versions of everything. Let's run an upgrade later!